sdk/attestation/azure-security-attestation/tests/test_azure_attestation_async.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import asyncio
+import functools
+import os
+
+from devtools_testutils import AzureTestCase, PowerShellPreparer
+from azure_devtools.scenario_tests.utilities import trim_kwargs_from_test_function
+from devtools_testutils.azure_testcase import _is_autorest_v3
+from devtools_testutils import AzureTestCase
+from azure.security.attestation.aio import AttestationClient
+
+AttestationPreparer = functools.partial(
+            PowerShellPreparer, "attestation",
+#            attestation_azure_authority_host='xxx',
+#            attestation_resource_group='yyyy',
+#            attestation_subscription_id='xxx',
+#            attestation_environment='AzureCloud',
+            attestation_policy_signing_key0='keyvalue',
+            attestation_policy_signing_key1='keyvalue',
+            attestation_policy_signing_key2='keyvalue',
+            attestation_policy_signing_certificate0='more junk',
+            attestation_policy_signing_certificate1='more junk',
+            attestation_policy_signing_certificate2='more junk',
+            attestation_serialized_policy_signing_key0="junk",
+            attestation_serialized_policy_signing_key1="junk",
+            attestation_serialized_policy_signing_key2="junk",
+            attestation_serialized_isolated_signing_key='yyyy',
+            attestation_isolated_signing_key='xxxx',
+            attestation_isolated_signing_certificate='xxxx',
+            attestation_service_management_url='https://management.core.windows.net/',
+            attestation_location_short_name='xxxx',
+            attestation_client_id='xxxx',
+            attestation_client_secret='secret',
+            attestation_tenant_id='tenant',
+            attestation_isolated_url='https://fakeresource.wus.attest.azure.net',
+            attestation_aad_url='https://fakeresource.wus.attest.azure.net',
+#            attestation_resource_manager_url='https://resourcemanager/zzz'
+        )
```

Comment:
I would recommend breaking this out into a separate file and importing it in each test file, reducing code 😃

Reply:
Tried, I'm getting "attempted relative import with no known parent package".

Reply:
An [example](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/tables/azure-data-tables/tests/test_retry.py#L34) in tables SDK. You don't need to do a `from .preparers import AttestationPreparer`, you can drop the '.'